src/pudl/extract/excel.py

```diff
@@ -85,17 +85,6 @@ def get_file_name(self, page, **partition):
         """Returns file name of given partition and page."""
         return self._file_name.loc[page, str(self._get_partition_selection(partition))]
 
-    def get_column_map(self, page, **partition):
-        """Return dictionary for renaming columns in a given partition and page."""
-        return {
-            v: k
-            for k, v in self._column_map[page]
-            .T.loc[str(self._get_partition_selection(partition))]
-            .to_dict()
-            .items()
-            if v != -1
-        }
-
```

Comment:
i moved this from the excel extractor to the generic extractor with the general idea that `process_raw` should actually do any renaming. This was one of several little migrations that needed to happen for that.

Reply:
Oops, perils of parallelism. We're doing similar things in slightly different ways, as I ran into the same issue in #3497.

Reply:
But I think this looks neat and yours will get merged in first, so I'll handle the conflict!